src/freenas/etc/netcli

```diff
@@ -76,6 +76,18 @@ from requests.packages.urllib3 import poolmanager
 from requests.packages.urllib3.connectionpool import HTTPSConnectionPool
 
 
+class FailoverIsEnabledException(Exception):
+    pass
+
+
+def ensure_failover_is_disabled():
+    if not notifier().is_freenas():
+        from freenasUI.failover.models import Failover
+        if Failover.objects.all().exists():
```

Comment:
Not that it matters much but you could probably save a query if you save Failover.objects.all() as `qs`